spec/models/service_template_spec.rb

```diff
@@ -1020,6 +1020,29 @@
         end
       end
 
+      context "with parent request id" do
+        let(:arg2) { {:initiator => 'control', :submit_workflow => true, :parent_id => 417} }
```

Comment:
Please don't hardcode ids in the specs because it breaks multi-region testing.  Two ways to do this are to create an actual record and use the record's id, or use a helper method that generates a fake id in the correct region (but only if you don't actually need the other record for anything).